io.github.chen08209.FlClash.yml

```diff
@@ -0,0 +1,45 @@
+id: io.github.chen08209.FlClash
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: AppRun
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=com.follow.clash
```

Comment:
why is this own-name needed ?

Reply:
is this actually a valid ID? Do they own follow.com ?

Reply:
> is this actually a valid ID? Do they own follow.com ?

I get this by run `flatpak run --log-session-bus --nosocket=session-bus io.github.chen08209.FlClash`
If I remove it, the software won't run.

```
$ flatpak run --log-session-bus --nosocket=session-bus io.github.chen08209.FlClash
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake AddMatch for org.kde.StatusNotifierWatcher
C3: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierWatcher
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C5: -> org.freedesktop.DBus fake wildcarded AddMatch for io.github.chen08209.FlClash
C6: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.io.github.chen08209.FlClash
C7: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
*SKIPPED*
B7: <- org.freedesktop.DBus return from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C11: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C12: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
C13: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierWatcher
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
*SKIPPED*
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
B12: <- org.freedesktop.DBus return from C11
*SKIPPED*
B13: <- org.freedesktop.DBus return from C12
*SKIPPED*
C14: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 com.follow.clash, policy: 0 (required 3)
*HIDDEN* (ping)
B14: <- org.freedesktop.DBus return from C13
*SKIPPED*
B15: <- (no sender) return from C14
*REWRITTEN*

** (FlClash:2): WARNING **: 10:24:16.328: Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
can you ask upstream if they own follow.com ? If that's the Dbus ID then the app should use `com.follow.Clash` ID as well.